From: Ben Hutchings <ben@decadent.org.uk>
Subject: cpt: Allow ext4 mounts

The existing checkpoint code apparently works for ext4 and most other
filesystems backed by a block device.  The OpenVZ patch for RHEL 6
allows it without any apparent need for specific support elsewhere in
the code.

--- a/kernel/cpt/cpt_files.h
+++ b/kernel/cpt/cpt_files.h
@@ -74,4 +74,5 @@
 	 strcmp(mnt->mnt_sb->s_type->name, "devpts") != 0 && \
 	 strcmp(mnt->mnt_sb->s_type->name, "proc") != 0 && \
 	 strcmp(mnt->mnt_sb->s_type->name, "sysfs") != 0 && \
-	 strcmp(mnt->mnt_sb->s_type->name, "binfmt_misc") != 0)
+	 strcmp(mnt->mnt_sb->s_type->name, "binfmt_misc") != 0 && \
+	 strcmp(mnt->mnt_sb->s_type->name, "ext4") != 0)
